web/pgadmin/tools/user_management/PgPermissions.py

```diff
@@ -0,0 +1,116 @@
+from flask_babel import gettext
+
+
+class AllPermissionTypes:
+    object_register_server = 'object_register_server'
+    tools_erd_tool = 'tools_erd_tool'
+    tools_query_tool = 'tools_query_tool'
+    tools_debugger = 'tools_debugger'
+    tools_backup = 'tools_backup'
+    tools_restore = 'tools_restore'
+    tools_import_export_data = 'tools_import_export_data'
+    tools_import_export_servers = 'tools_import_export_servers'
+    tools_search_objects = 'tools_search_objects'
+    tools_storage_manager = 'tools_storage_manager'
+    tools_maintenance = 'tools_maintenance'
+    tools_schema_diff = 'tools_schema_diff'
+    tools_grant_wizard = 'tools_grant_wizard'
+    storage_add_folder = 'storage_add_folder'
+    storage_remove_folder = 'storage_remove_folder'
+
+
+class AllPermissionCategories:
+    object_explorer = 'Object Explorer'
+    tools = 'Tools'
+    storage_manager = 'Storage Manager'
+
+
+class PgPermissions:
```

Comment:
class name should be renamed to "PgAdminPermissions" as "PgPermissions" seems like PostgreSQL permissions.